src/app/ApplicationTemplate.Shared.Views/App.xaml

```diff
@@ -12,9 +12,9 @@
 				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
 				
 				<!-- Uno material -->
```

Comment:
I think you can override the fonts too. That way you can probably avoid redefining all the styles that involve text.